app/services/project_data_service.rb

```diff
@@ -121,7 +121,7 @@ def report_runtime_version(dependency)
         "runtime_version:#{dependency.version == 'unknown version' ? 'none' : dependency.version.delete('^0-9.')}",
         "project:#{@project.name}",
         "criticality:#{@project.criticality}",
-        "tags:#{@project.tags&.none? ? 'none' : @project.tags.join(':')}"
+        "tags:#{@project.tags&.blank? ? 'none' : @project.tags.join(':')}"
```

Comment:
```suggestion
        "tags:#{@project.tags.blank? ? 'none' : @project.tags.join(':')}"
```

Sorry should have been more clear. Now we _don't_ want the nil-safe navigator. In case of `nil` (which responds to `blank?`), we want this to evaluate to `true` and return `"none"`.

Reply:
Not your bad, I should have read the code more closely. FIxed in [3e75172](https://github.com/artsy/horizon/pull/586/commits/3e7517213e558aa6e58143c142c5a3515e4f55a6)